### ISSUE TITLE
[Feat] Add action roll specific enforcing of roll mode overrride

### ISSUE THREAD

**Issue by NekroDarkmoon** [Pjb518/FoundryVTT-Level-Up-Official#331]
### Tasks
- [ ] Add `defaultRollMode` config for rolling ability, skill, tool, attack rolls.
- [ ] Update handlers to account for these overrides. (Double check if action rolls are actually hooked into rollOverrides)

> There are a lot of features that dictate making a roll with advantage or disadvantage, and there isn't a way to configure that currently. I add a note in the label, to remind the DM/player, but it would be nice to have it built in.
- nchezrony

**Comment by NekroDarkmoon** [comment]
Blocked by #255
